pkg/mkimage-raw-efi/parse-interactive-installer-conf

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+CMDLINE=$(cat /proc/cmdline)
+INSTALLER_CONF="/opt/interactive-installer/installer.json"
+
+# Parse the installer JSON using jq and create an associative array
+declare -A assoc_array
+while IFS="=" read -r key value; do
+    if [ "$value" == "" ];
+    then
+        continue
+    fi
+    key=$(echo "$key" | tr '[:upper:]' '[:lower:]')
+    assoc_array["$key"]="$value"
```

Comment:
Using arrays here and in other places implies the use of bash, so far we have based all of our scripts in sh (and we don't even install bash), that's not a big deal include bash, but in this case you could just store the keypairs KEY=VAR into a file and then use a small regex to parse the file in the code below.... while the full handling of config files is not implemented in the installer....